### DISCUSSION TITLE
stop mode by likysoapp

### DISCUSSION THREAD

**Discussion by rubenwilson1** [AzuraCast/AzuraCast#4767] [General]
good morning I raise several queries if someone has found a solution for them, or our esteemed developer proposes something to be able to solve it,

When we work in radio and we have a 24-hour programming with advertising, jingles and canned files and live programs, we have both immediate and delayed modes handling, but we have a problem and that is that when we handle immediate mode the main playlist pauses and it does not stop what this causes that when the event ends we will continue with the same song and in the same segment where we stayed,
This would be solved by stopping the playlist. Is there any way to do it? I understand that there is a code for likysoapp but as when restarting it overwrites the initial configuration as it would be erased, my question will you solve this? It is important and it is also urgent because a radio cannot be every time event that is played, it cannot be pausing the list because when the pause is removed it plays where it left off and that is wrong.

the same happens when a source is connected or disconnected,
the list is paused
that as for the first question,
and as for the second, is it possible to reproduce a url, delayed? and how would it be done?
thanks for your answers and hopefully we all find a solution
because that enhances the professionalism of a radio station in the cloud.

greetings colleagues!

**Comment by rubenwilson1** [comment]
### I see that no one has answered this comment? Is the request not going to be taken into account?

**Comment by BusterNeece** [comment]
I believe the problem you're having is that changes to your Liquidsoap configuration aren't being kept during updates or restarts.

If you provide custom Liquidsoap configuration via Station Management > Utilities > Edit Liquidsoap Configuration, that custom configuration will be preserved even during updates and station restarts.

That should solve your problem, hopefully!

**Comment by rubenwilson1** [comment]
you answer part of the problem but you leave the rest unanswered,
How to make the playlist not pause when I play an instant event or when I go on air, I want the playlist to stop and play the next song when the event or connection with a dj ends

**Reply by BusterNeece** [reply]
This is a different issue that we've been working on for several years. We hope to have an update for this before we tag our next stable release.

**Comment by rubenwilson1** [comment]
And is it possible that events can also be programmed by seconds as well as by minutes and by hours?

**Reply by BusterNeece** [reply]
It's hard for us to keep track if you add a bunch of questions into one single discussion or issue. It would be best if you joined us in our [Discord](https://discord.gg/azuracast) to ask those questions.

**Comment by rubenwilson1** [comment]
I would join what happens is that I do not speak English write yes, because I can write through translations but I do not speak it.
